packages/hash/shared/src/blocks.ts

```diff
@@ -234,7 +234,7 @@ export const fetchBlock = async (
 export const paragraphBlockComponentId =
   "https://blockprotocol.org/blocks/@hash/paragraph";
 
-const textBlockComponentIds = new Set([
+export const textBlockComponentIds = new Set([
```

Comment:
Better to use `isHashTextBlock` than to export this set

Reply:
https://github.com/hashintel/hash/pull/1014/commits/0f6b38a572f46b6190083c4edca2231134a1383f